pulp_rpm/app/models.py

```diff
@@ -204,6 +254,20 @@ class SrpmContent(RpmContent):
 
     TYPE = 'srpm'
 
+    @classmethod
+    def is_srpm(cls, package):
```

Comment:
I think if we are going to add methods like this, we should take everything out of RpmContent and SrpmContent and make an abstract RpmBase model that they both inherit from.

Otherwise, I'm not sure how useful this method is, because all SrpmContent should, hopefully, be Srpms by definition.